src/routes/applications.js

```diff
@@ -0,0 +1,11 @@
+const router = require('express').Router();
+const auth = require('./auth');  // assuming you have an auth middleware to authenticate users
```

Comment:
we'll add authentication functionality on the next tasks. for now focus on CRUD operations